cms/static/js/views/settings/main.js

```diff
@@ -24,6 +24,7 @@ var DetailsView = ValidatingView.extend({
     initialize : function() {
         this.fileAnchorTemplate = _.template('<a href="<%= fullpath %>"> <i class="icon fa fa-file"></i><%= filename %></a>');
         // fill in fields
+        this.$el.find("#course-language").val(this.model.get('language'));
```

Comment:
Have you verified that this doesn't error if about_page_editable is False (meaning there is no element with ID course-language on the page)? 

You might need to add a unit test around this case.